src/coq_elpi_HOAS.ml

```diff
@@ -1105,6 +1108,12 @@ let get_options ~depth hyps state =
       let _, rd, gl = reduction_flags.Elpi.API.Conversion.readback ~depth state t in
       assert (gl = []);
       Some rd in
+  let keeping_algebraic_universes s =
+    if s = Some "default" then None
+    else if s = Some "keep-alguniv" then Some true
+    else if s = Some "purge-alguniv" then Some false
+    else if s = None then None
```

Comment:
maybe a bool would just do it then

Reply:
What was the rationale  for choosing `bool option` over `bool` in almost all other attributes? E.g. `coq:keepunivs`, `coq:locality`, etc

Reply:
I did update it. My point was to have an option carrying a bool, not a string.